security/security-design/fn-datasafe-dbaudit-to-oci-logging/terraform/function/oci-datasafe-audit-to-logging/build-fn.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+. ./set-local-vars.sh
+
+mkdir dist
+rsync -aC --exclude node_modules --exclude dist --exclude '.git' --exclude '*.sh' --exclude test --exclude '.gitignore' . ./dist
+cd dist
+fn -v deploy --app $FN_APP_NAME
+cd ..
+#copy the updated func.yaml back for versioning?
```

Comment:
Use proper English grammar and punctuation, i.e., start a sentence with a capital letter, end a sentence with punctuation, have a space between comment character and first letter of the text, etc.

Reply:
Correct typos in file